.buildkite/pipeline-for-tags.yml

```diff
@@ -162,5 +162,5 @@ steps:
    soft_fail: true # No artifacts to download if all the bottles are already built
```

Comment:
The branch name should have the issue number in it, see [our guidelines](https://www.notion.so/serokell/Where-and-how-to-commit-your-work-58f8973a4b3142c8abbd2e6fd5b3a08e?pvs=4#c6e4c35d7aa74d1ca0b43ae049a3c847).

AFAIK you'd have to close this PR and open another one, so I think that for this small change it's not worth the trouble, but please name the branch correctly next time.